app/src/main/java/com/example/hungrywolfs/MainActivity.kt

```diff
@@ -2,10 +2,26 @@ package com.example.hungrywolfs
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.setupWithNavController
+import com.google.android.material.bottomnavigation.BottomNavigationView
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var navController: NavController
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
         setContentView(R.layout.activity_main)
+
```

Comment:
A bit too many empty lines, even though at times it helps with readability, it becomes tiresome if we add an empty line after every code line